static/src/javascripts/projects/membership/payment-form.js

```diff
@@ -34,7 +34,7 @@ define([
             ERROR_CARD_EXPIRY: 'js-error--card-expiry',
             HIDE: 'is-hidden',
             PAYMENT_ERRORS: 'js-payment-errors',
-            FORM_SUBMIT: 'js-submit-input',
+            FORM_SUBMIT: 'submit-input',
```

Comment:
I would say add the `js-submit-input` class to the HTML rather than amending the JavaScript